cachito/web/migrations/env.py

```diff
@@ -79,13 +81,22 @@ def process_revision_directives(context, revision, directives):
             target_metadata=target_metadata,
             process_revision_directives=process_revision_directives,
             render_as_batch=True,
+            include_object=include_object,
             **current_app.extensions["migrate"].configure_args,
         )
 
         with context.begin_transaction():
             context.run_migrations()
 
 
+# Refer to: https://alembic.sqlalchemy.org/en/latest/cookbook.html#don-t-generate-any-drop-table-directives-with-autogenerate
+def include_object(object, name, type_, reflected, compare_to):
+    if type_ == "table" and reflected and compare_to is None:
```

Comment:
Would it make sense to explicitly ignore only the 3 affected tables - `"package", "request_package", "request_dependency"`?